backend/src-common/pom.xml

```diff
@@ -37,11 +37,24 @@
         </plugins>
     </build>
     <dependencies>
+    <!-- https://mvnrepository.com/artifact/com.googlecode.json-simple/json-simple -->
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
+            <artifactId>json-simple</artifactId>
+            <version>1.1.1</version>
+            <scope>compile</scope>
+        </dependency>
         <dependency>
             <groupId>org.spdx</groupId>
             <artifactId>spdx-tools</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.spdx</groupId>
+            <artifactId>tools-java</artifactId>
+            <version>1.0.2</version>
```

Comment:
Please add the versions on root pom.xml, and the dependencies on DependencyManagement.